website/templates/nav.mako

```diff
@@ -48,19 +48,22 @@
             <li class="navbar-donate-button"><a id="navbar-donate" data-bind="click: trackClick.bind($data, 'Donate')" href="https://cos.io/donate">Donate</a></li>
             % if user_name and display_name:
             <li class="dropdown">
-            <a class="dropdown-toggle btn-link" data-toggle="dropdown" role="button" aria-expanded="false" aria-label="Toggle auth dropdown">
-                <span class="osf-profile-image">
-                    <img src="${user_profile_image}" alt="User profile image">
-                </span> ${display_name}
-                <span class="caret"></span>
-            </a>
+                <a class="dropdown-toggle btn-link" data-toggle="dropdown" role="button" aria-expanded="false" aria-label="Toggle auth dropdown">
+                    <div class="osf-profile-image">
+                        <img src="${user_profile_image}" alt="User profile image">
+                    </div>
+                    <div class="nav-profile-name">
+                            ${display_name}
```

Comment:
Truncating the display name with CSS obviates the need to truncate the actual value here. If the value is truncated, but still too long, you could end up with something weird like `This is a Very...Long Name That...`

Reply:
I'm not sure if there is a way to easily get the non-truncated Display Name.

Reply:
I've stopped the truncation, this whole time I didn't know we did this server-side.